ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -88,13 +89,13 @@ def main(argv=sys.argv[1:]):
     if args.xunit_file:
         start_time = time.time()
 
-    files = get_files(args.paths, extensions)
-    if not files:
-        print('No files found', file=sys.stderr)
+    compilation_dbs = get_compilation_db_files(args.paths)
+    if not compilation_dbs:
+        print('No compilation database files found', file=sys.stderr)
         return 1
 
     bin_names = [
-        'clang-tidy',
+        # 'clang-tidy',
```

Comment:
Only use `clang-tidy` 6.0 for now, because it is the most ubiquitous, the default version of `clang-tidy` on Ubuntu Bionic seems to be a moving target, and the parsing of linter warning/error messages has very recently been made difficult by an update to the default version of `clang-tidy` on Ubuntu Bionic. This is also consistent with instructions in the README that's part of this pull request.

Reply:
The master branch is not targeting Ubuntu Bionic anymore but Ubuntu Focal. And the default version of `clang-tidy` in Focal is 10.0. So I am not sure it makes sense to target version 6 here...

Reply:
Will the `master` branch be backported to the branches that target Ubuntu Bionic, for example for the Dashing distro? Since the `clang-tidy-6.0` apt package is valid for both Ubuntu Bionic and Focal, I think we should stick with `clang-tidy` 6.0 for now, then maybe backport, then target `clang-tidy` 10.0 (along with other features that I have in mind for this package).

Reply:
Changes need to be explicitly backport through PRs. It won't happen automatically.

Using version 10 on Focal should happen before the upcoming freeze date. I don't mind which way that happens (target version 10 in this PR and then make a backport PR for Eloquent / Dashing which uses version 6) orland this PR targeting 6 and followingup with a PR to switch to version 10 with the intention to only backport the first.

Reply:
Let's land this PR targeting `clang-tidy` 6.0, with a follow-up PR to switch to `clang-tidy` 10.0.